crates/bin/src/args.rs

```diff
@@ -298,14 +298,15 @@ impl Default for RateLimit {
 
 /// Strategy for installing the package
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, ValueEnum, EnumCount)]
+#[repr(u8)]
 pub enum Strategy {
     /// Attempt to download official pre-built artifacts using
     /// information provided in `Cargo.toml`.
-    CrateMetaData,
+    CrateMetaData = 0,
```

Comment:
This is implied: there's no need to do it.

From [ref](https://doc.rust-lang.org/reference/items/enumerations.html):

> If the first variant in the declaration is unspecified, then it is set to zero. For every other unspecified discriminant, it is set to one higher than the previous variant in the declaration.

Reply:
Done